build/style_test.go

```diff
@@ -803,4 +803,6 @@ func TestStyle(t *testing.T) {
 			}
 		}
 	})
+
+	t.Run("TestURLLiveness", func(t *testing.T) { testURLLiveness(t, pkg) })
```

Comment:
Could just make it a 1st class test and avoid the empty stub in the common case?